pkg/tanka/inline.go

```diff
@@ -25,7 +25,20 @@ func (i *InlineLoader) Load(path string, opts LoaderOpts) (*v1alpha1.Environment
 	if opts.Name != "" {
 		opts.JsonnetOpts.EvalScript = fmt.Sprintf(SingleEnvEvalScript, opts.Name)
 	}
+	return i.load(path, opts)
+}
+
+func (i *InlineLoader) Peek(path string, opts LoaderOpts) (*v1alpha1.Environment, error) {
```

Comment:
@dylwylie As you pointed out in DM, this indeed only affects when exporting a single (set of) environments, it works differently when using `--recursive`, so the performance issue is still a problem when doing `tk export --recursive`. Looking into that now.